### PR TITLE
GitHub Actions tutorial

### PR DESCRIPTION
**Pull request by karensawrey**
- Draft pull request to make sure the commits don't end up in a void _again_.

### CODE REVIEW

File: pages/tutorials/github_actions.md.erb [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+## Testing for development
+
+To run the test workflow locally, you can use [act](https://github.com/nektos/act, which will run it the same way it does on GitHub:
+
+```
+act
+```
+
+## Releasing
+Create a new GitHub release. The version numbers in the readme will be automatically updated.
```
Comment (plaindocs) [inline pages/tutorials/github_actions.md.erb:80]: This is probably not meant to be here?

**Review by plaindocs — CHANGES_REQUESTED** [review]
This is a great start, but it's not ready to merge yet! 
Let's look at it next week :muscle: :+1:

**Comment by karensawrey** [comment]
@plaindocs thanks! Just the reality check I needed!

File: data/nav.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 - - name: 'Pipelines'
     path: 'tutorials/getting-started'
+    children:
```
[the remark this answers is not in the document]
Comment (karensawrey) [reply data/nav.yml:4]: Nope, this is intentional, as my syntax checker caught an extra space, so I tweaked it a bit

File: pages/tutorials/github_actions.md.erb (hunk 1 above)
Comment (karensawrey) [reply pages/tutorials/github_actions.md.erb:80]: Now that I think about it, the contributing section is definitely overkill in a tutorial :D

File: pages/tutorials/github_actions.md.erb [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+[GitHub Actions](https://github.com/features/actions) feature helps automate GitHub-based workflows. You can use a [Buildkite GitHub Action](https://github.com/actions) for triggering a build on a Buildkite pipeline.
+
+{:toc}
+
+The Buildkite GitHub Action allows you to:
+
+* Create builds in Buildkite pipelines and set `commit`, `branch`, `message`.
+* Save the build JSON response to `${HOME}/${GITHUB_ACTION}.json` for downstream actions.
+
+<div class="Docs__note">
```
[the remark this answers is not in the document]
Comment (karensawrey) [reply pages/tutorials/github_actions.md.erb:12]: Do you mean, to turn Prerequisites into a separate small H2 1-paragraph section?

File: pages/tutorials/github_actions.md.erb [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+```yml
+- name: Trigger Buildkite Pipeline
+  uses: buildkite/trigger-pipeline-action@v1.2.0
+```
+{: codeblock-file="pipeline.yml"}
+
+<%= image "trigger-buildkite-pipeline.png", width: 2630/2, height: 1692/2, alt: "Trigger Buildkite Pipeline GitHub Action on GitHub" %>
+
+## Creating and configuring the workflow for Buildkite GitHub Actions
```
[the remark this answers is not in the document]
Comment (karensawrey) [reply pages/tutorials/github_actions.md.erb:33]: ✔️

File: pages/tutorials/github_actions.md.erb [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+1. Create a [Buildkite API access token](/docs/apis/rest-api#authentication) with `write_builds` [scope](/docs/apis/managing-api-tokens#token-scopes), and save it to your GitHub repository’s Settings in Secrets.
+
+2. Configure your GitHub Actions workflow with the details of the pipeline to be triggered, and the settings for the build.
+
+For example, the following workflow creates a new Buildkite build on every commit (change `my-org` to the slug of your org):
+
+```
```
[the remark this answers is not in the document]
Comment (karensawrey) [reply pages/tutorials/github_actions.md.erb]: Good old YAML.

File: pages/tutorials/github_actions.md.erb [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+2. On the [Trigger Buildkite Pipeline](https://github.com/marketplace/actions/trigger-buildkite-pipeline) page, click "Use latest version".
+
+3. Copy and paste the following snippet into your .yml file.
+
+```yml
+- name: Trigger Buildkite Pipeline
+  uses: buildkite/trigger-pipeline-action@v1.2.0
+```
+{: codeblock-file="pipeline.yml"}
```
[the remark this answers is not in the document]
Comment (karensawrey) [reply pages/tutorials/github_actions.md.erb]: it IS in  `.github/workflows` though, in the yaml file of an arbitrary name that corresponds to the name of the project

**Comment by lachlanhardy** [comment]
@karensawrey  Heyo! Do you reckon this is mergeable or should the Onboarding team take another look at it?

**Comment by karensawrey** [comment]
@lachlanhardy everything should go through the Onboarding team, for sure.

@mbelton-buildkite please take a look when you have time.

**Comment by dannymidnight** [comment]
I've resolved the conflicts and brought this branch back to life. There still remains some linting errors but perhaps not far off merging assuming content still holds up.

**Comment by mbelton-buildkite** [comment]
I believe the content for this needed an update. @lizrabuya was going to take a look as part of [this escalation](https://3.basecamp.com/3453178/buckets/1713315/card_tables/cards/6215864847). How's that going @lizrabuya?

**Comment by lizrabuya** [comment]
> I believe the content for this needed an update. @lizrabuya was going to take a look as part of [this escalation](https://3.basecamp.com/3453178/buckets/1713315/card_tables/cards/6215864847). How's that going @lizrabuya?

Thanks for the bump on this @mbelton-buildkite . This has been put on hold for too long. I'll put this in our cycle planning to get it scheduled.

**Review by MelissaKaulfuss — CHANGES_REQUESTED** [review]
Looking great! ✨

Just a few little things 🤏(some of them you might want to ignore).

**Comment by lizrabuya** [comment]
@MelissaKaulfuss , thanks for the inputs. I have incorporated all the changes requested into this one commit [`6770d62`](https://github.com/buildkite/docs/pull/1682/commits/6770d620f62a796c95e749010eaa09188d44e942).

**Review by MelissaKaulfuss — APPROVED** [review]
Looking good @lizrabuya! 😊

**Comment by MelissaKaulfuss** [comment]
@dannymidnight we can merge these changes 🚀

**Pull request merged**